src/test/test_dir.c

```diff
@@ -6573,6 +6573,87 @@ test_dir_dirserv_load_fingerprint_file(void *arg)
   dirserv_free_fingerprint_list();
 }
 
+#define RESET_FP_LIST(list) STMT_BEGIN \
+    dirserv_free_fingerprint_list(); \
+    authdir_init_fingerprint_list(); \
+    list = authdir_return_fingerprint_list(); \
+  STMT_END
+
+#define FP_REJECT 4
+
+static void
+test_dir_dirserv_router_get_status(void *arg)
```

Comment:
What happens when dirserv_router_get_status() is called on an empty fingerprint list?
What should happen?

Reply:
I have added this test. `dirserv_router_get_status()` returns 0 in this case.

Reply:
c2d60099fd1a7291eb05292ae45665632fa2abf0